rollup/cmd/gas_oracle/app/app.go

```diff
@@ -92,7 +92,9 @@ func action(ctx *cli.Context) error {
 	}
 	// Start l1 watcher process
 	go utils.LoopWithContext(subCtx, 10*time.Second, func(ctx context.Context) {
-		number, loopErr := butils.GetLatestConfirmedBlockNumber(ctx, l1client, cfg.L1Config.Confirmations)
+		// Fetch the latest block number to decrease the delay when fetching gas prices
+		// Use latest block number - 1 to prevent frequent reorg
+		number, loopErr := butils.GetLatestConfirmedBlockNumber(ctx, l1client, 2)
```

Comment:
why use `2` here?

Reply:
In the implementation of `GetLatestConfirmedBlockNumber`, it's equivalent to `rpc.LatestBlockNumber` then `loopErr = l1watcher.FetchBlockHeader(number-1);`.

Reply:
fixed in [e66b5c6](https://github.com/scroll-tech/scroll/pull/920/commits/e66b5c6d3a1c0a6a9531b29aa8089cd2c4a8531e)